contracts/governance/timelock-controller/tests.rs

```diff
@@ -0,0 +1,384 @@
+#[cfg(test)]
+#[brush::contract]
+mod tests {
```

Comment:
Looks like integration tests. Can we put all integ tests in the tests folder? Cargo considers all integ tests are there. And optionally we could split this file on multiple and give them a meaningful name according to what feature is tested